src/main/bash/sdkman-home.sh

```diff
@@ -35,5 +35,10 @@ function __sdk_home() {
 		return 1
 	fi
 
-	echo -n "${SDKMAN_CANDIDATES_DIR}/${candidate}/${version}"
+		
+	if [[ "$sdkman_home_newline" == "false" ]]; then
+    	echo -n "${SDKMAN_CANDIDATES_DIR}/${candidate}/${version}"
```

Comment:
In POSIX sh, echo flags are undefined. [SC3037](https://github.com/koalaman/shellcheck/wiki/SC3037)
Replace by:
```sh
printf %s "${SDKMAN_CANDIDATES_DIR}/${candidate}/${version}"
```